src/common/calendar/TimePickerCalendar.js

```diff
@@ -37,15 +37,33 @@ class TimePickerCalendar extends Component {
 
   state = {
     viewType: 'timeGridWeek',
-    selected: calendarUtils.getDefaultSelectedTimeRange(this.props.edittingReservation)
+    selected: calendarUtils.getDefaultSelectedTimeRange(this.props.edittingReservation),
+    header: {
+      left: 'prev,next today',
```

Comment:
Don't really know how this really works, but I assume it should be:
```suggestion
      left: 'prev,next,today',
```

Reply:
They are designed to be like that for small gap between <prev,next> and today button. Check the calendar render